README.md

```diff
@@ -0,0 +1,96 @@
+# Net Agent
+
+[![Crates.io](https://img.shields.io/crates/v/net-agent.svg)](https://crates.io/crates/net-agent)
+[![License](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+
+`net-agent` is a binary crate that allows you to capture network traffic and load it to [my.netstalker.io](https://my.netstalker.io) for further analysis.
+
+## Installation
+
+To install `net-agent`, use Cargo:
+
+```shell
+cargo install net-agent
+```
+
+## Requirements
+Before running `net-agent`, make sure you have the necessary dependencies installed for your operating system:
+
+### Linux
+
+- Ubuntu/Debian:
+    ```shell
+    sudo apt-get install libpcap-dev
+    ```
+
+- Fedora:
+    ```shell
+    sudo dnf install libpcap-devel
+    ```
+- Arch:
+    ```shell
+    sudo pacman -S libpcap
+    ```
+### macOS
+
+- Install Xcode Command Line Tools::
+    ```shell
+    xcode-select --install
+    ```
+- Install libpcap using Homebrew::
+    ```shell
+    brew install libpcap
+    ```
+
+### Windows
+
+- Download and install Npcap (compatible with Windows 7, 8, 10).
+
+## Usage
+
+You can run `net-agent` with the following command-line options:
+
+```shell
+net-agent [OPTIONS]
+```
+
+The available options are:
+
+- `-D, --device-name <DEVICE_NAME>`: Specify the network device to capture traffic from.
+- `-N, --number-packages <NUMBER_PACKAGES>`: Specify the number of packages to capture.
+- `-B, --buffer-size <BUFFER_SIZE>`: Specify the buffer size for capturing packages.
+- `O, --output-directory <OUTPUT_DIRECTORY>`: Specify the output directory for captured traffic (default: "output").
+
+Note: The `-C`, `--config-file` option conflicts with all other options. If specified, the configuration will be loaded from the specified file.
+
+## Configuration
+
+```toml
+device_name = "eth0"
+number_packages = 1000
+buffer_size = 2048
+output_directory = "captured_traffic"
+```
+
+To use the configuration file, pass the `-C`, `--config-file` option with the path to the file.
```

Comment:
I will like it more, if we place *things at line 75* before config code. And I would love to see template, not concrete examples.